modin/pandas/test/test_series.py

```diff
@@ -396,16 +396,20 @@ def test___gt__(data):
     inter_df_math_helper(modin_series, pandas_series, "__gt__")
 
 
-@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
-def test___int__(data):
-    modin_series, pandas_series = create_test_series(data)
-    try:
-        pandas_result = int(pandas_series[0])
-    except Exception as err:
-        with pytest.raises(type(err)):
-            int(modin_series[0])
-    else:
-        assert int(modin_series[0]) == pandas_result
+@pytest.mark.parametrize("count_elements", [0, 1, 10])
+def test___int__(count_elements):
+    eval_general(
+        *create_test_series(test_data["float_nan_data"]),
```

Comment:
1. To avoid returning `RecursionError` in the future.
2. Now a float number is being tested, I suppose it is not necessary to test np.nan separately.

Reply:
> 2\. Now a float number is being tested, I suppose it is not necessary to test np.nan separately.

Why? I think we should test it too.

Reply:
According to the current implementation it seems redundant and the implementation change seems unlikely.